### PR TITLE
Fix contains pointer bug 

### PR DESCRIPTION
**Pull request by liajoy**
seems multiply the viewportTransform can solve this problem.
reference: https://github.com/fabricjs/fabric.js/issues/5141

### CODE REVIEW

File: src/canvas.class.js [hunk 1]
```diff
@@ -493,10 +493,12 @@
      * @private
      */
     _normalizePointer: function (object, pointer) {
-      var m = object.calcTransformMatrix(),
+      var vpt = this.viewportTransform,
+          m = object.calcTransformMatrix(),
           invertedM = fabric.util.invertTransform(m),
+          matrix = fabric.util.multiplyTransformMatrices(vpt, invertedM),
           vptPointer = this.restorePointerVpt(pointer);
```
[the remark this answers is not in the document]
Comment (liajoy) [reply src/canvas.class.js]: @asturur 
the invertedM doesn't multiply the viewportTransform?
the test demo: https://jsfiddle.net/8ea1ux97/14/
Maybe do some unit tests is better.

[the remark this answers is not in the document]
Comment (asturur) [reply src/canvas.class.js]: yes bu the restorePointVpt removes it.
At this point we could remove both lines.

[the remark this answers is not in the document]
Comment (liajoy) [reply src/canvas.class.js]: yes these lines can be removed. If removed, there is a simple demo:
if viewPortTransform = [0.5, 0, 0, 0.5, 0, 0]
if invertedM = [1, -0, -0, 1, -100, -100]
the correct matrix = [1, -0, -0, 1, -100 * 0.5, -100 * 0.5]  (not [0.5, -0, -0, 0.5, -100 * -0.5, -100 * 0.5], the pointer just need translate no scale)
In this case, we should set a new matrix. so seems restorePointVpt first then multiply is more suitable?

**Comment by liajoy** [comment]
there is something missing before. The key point of reproduce is call setCoords of object which makes acoords of object with viewport state. So _normalizePointer should multiply viewport Transform.

**Comment by asturur** [comment]
@liajoy thanks for bringing this up. The code works, the problem was probably in how the group was created.

**Pull request closed without merging**